### ISSUE TITLE
Consider dropping support for netstandard1.3

### ISSUE THREAD

**Issue by ViktorHofer** [HangfireIO/Hangfire#1851]
Most libraries already target .NET Standard 2.0 or a supported platform that support .NET Standard 2.0 like

.NET Core 2.0 and above
.NET Framework 4.6.1 and above
UAP10.0.16299

I would suggest to consider dropping support for .NET Standard 1.3 and older .NET Framework versions in favor of a broader API set a simplified dependency graph. Other open source repositories are dropping support for older targets as well, i.e. xunit v3. I would be willing to send a PR but wanted to open an issue to discuss this first.

**Comment by odinserj** [comment]
Yes, it will be dropped, but only with the next version that allows breaking changes, e.g. Hangfire 2.0. And currently it is possible to implement a lot of features without introducing any breaking changes anyway. `netstandard1.3` target doesn't add significant complexity, so we can postpone its removal.

**Comment by ViktorHofer** [comment]
Thanks for sharing your stance. Any expectation when the next major version of Hangfire will be released? Just curious.

**Comment by odinserj** [comment]
[1.8.0-beta1](https://github.com/HangfireIO/Hangfire/releases/tag/v1.8.0-beta1) and [1.8.0-beta2](https://github.com/HangfireIO/Hangfire/releases/tag/v1.8.0-beta2) already available, no need in breaking changes yet – I don't think "move fast and break things" is the correct approach here. The presence of `netstandard1.3` target doesn't make any harm for us.
